pkgs/development/python-modules/pelican/default.nix

```diff
@@ -48,6 +48,11 @@ buildPythonPackage rec {
     nose
   ];
 
+  patches = [
+    # fix for updated pygments version. Remove when updating to new version.
+    ./pygments-tests.patch
```

Comment:
Is this taken from a pull request? If so it would be nice to use `fetchpatch` on the commit url instead of adding the patch to nixpkgs.

Reply:
Just saw the other comments above.